tooling/fixtures/fixture_data/bls12-381-sha-256/proof/proof001.json

```diff
@@ -0,0 +1,14 @@
+{
+  "caseName": "single message signature, message revealed proof",
+  "signerPublicKey": "b65b7cbff4e81b723456a13936b6bcc77a078bf6291765f3ae13170072249dd7daa7ec1bd82b818ab60198030b45b8fa159c155fc3841a9ad4045e37161c9f0d9a4f361b93cfdc67d365f3be1a398e56aa173d7a55e01b4a8dd2494e7fb90da7",
+  "header": "11223344556677889900aabbccddeeff",
+  "presentationHeader": "bed231d880675ed101ead304512e043ade9958dd0241ea70b4b3957fba941501",
+  "revealedMessages": {
+    "0": "9872ad089e452c7b6e283dfac2a80d58e8d0ff71cc4d5e310a1debdda4a45f02"
```

Comment:
All the revealed messages are numbered using a 0-based counter in the tests. My implementation uses a 1-based one, matching the sub-indices in the spec. This could have interop implications when these proofs are serialized, so we should call out the expected behavior. My personal vote: use a one-based counter.

Reply:
Good point have captured notes of this in #225 and #7